dockers/docker-fpm/Dockerfile

```diff
@@ -12,6 +12,9 @@ COPY deps/fpmsyncd /usr/local/bin/
 ## Get Quagga
 RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; }; dpkg_apt /deps/quagga_*.deb
 
+COPY daemons /etc/quagga
+COPY zebra.conf /etc/quagga
+COPY bgpd.conf /etc/quagga
```

Comment:
This file is switch specific, and should not statically embedded into docker image. We may use Ansible template to construct one and mount the file when running the docker container.